src/routes/+page.svelte

```diff
@@ -53,214 +47,115 @@
-		</p>
-
-		<h2>Wasn't your previous name “Chaos”?</h2>
-		<p>
-			Yep! When the project started in 2020 it was under the name “Chaos Source” with contributors
-			organized under the umbrella of “Chaos Initiative” and the P2:CE team known as “Chaos
-			Studios”. This naming was chosen in very early days of the project, and while it was okay for
-			a time, in 2023 we decided to optimize our branding with input from a wider range of project
-			members and decided on <strong>Strata Source</strong>.
-		</p>
-	</div>
+	<h2>What is Strata Source?</h2>
+	<p>
+		Strata Source began as the Source Engine branch behind Portal 2: Community Edition (P2:CE) in
+		2020. In late 2020, Momentum Mod joined forces with the P2:CE team with the goal of
+		collaborating on this new branch. The aim of Strata is to foster collaboration between licensed
+		projects. Today, a team of almost two dozen developers contribute to Strata Source, with three
+		licensed projects using the engine.
+	</p>
+	<h2>How can I use Strata?</h2>
+	<p>
+		Strata is only accessible as part of the licensed games built using it; we do not provide any
+		publicly available SDK. Any licensed Source engine project is welcome to contact us regarding
+		collaboration.
+	</p>
+	<p>
+		If you’re interested in making a Portal 2 mod on Strata, check out <a
+			href="https://portal2communityedition.com/">P2:CE</a
+		>
+		or <a href="https://www.portalrevolution.com/">Portal: Revolution.</a>
+	</p>
+	<h2>Can Strata Help my Project Get Licensed?</h2>
+	<p>
+		<b>No.</b> Licensing is done entirely by Valve. We cannot license projects and don’t provide any
+		source code to unlicensed projects. We cannot influence Valve’s licensing decisions and don’t
+		offer legal advice. See Valve’s documentation on Source engine licensing:
+		<a href="https://partner.steamgames.com/doc/sdk/uploading/distributing_source_engine"
+			>Distributing Source Engine Games / Mods</a
+		>.
+	</p>
+	<h2>Can I use Strata’s Logo or Branding?</h2>
+	<p>
+		Please don’t use our logo or branding to represent your project. <!--If you are developing a P2:CE mod, see their guidelines on using P2:CE branding (LINK HERE).-->
+	</p>
+	<p>
+		You are welcome to use our logo and branding in material about Strata itself, such as videos or
+		articles. See our branding page: <a href="https://stratasource.org/branding">Strata Branding</a
+		>.
+	</p>
+	<h2>Why not Source 2?</h2>
+	<p>
+		We do not have access to Source 2 and are not licensed to use Source 2. All of the games
+		currently using Strata are closely tied to existing Source 1 content, so Source 2 would be a
+		poor choice for us.
+	</p>
+	<p>
+		We do have some Source 2 features, such as the Panorama UI system, that were backported by Valve
+		to CS:GO. However, Strata is still almost entirely Source 1. Whilst Source 1 does contain
+		considerable tech-debt, we’ve been able to improve modernize it considerably, and we think its
```

Comment:
```suggestion
		considerable tech-debt, we’ve been able to improve and modernize it considerably, and we think its
```

Reply:
Lol, this was like the one thing that I wrote. Fixed!